### DISCUSSION TITLE
Include systems to several groups

### DISCUSSION THREAD

**Discussion by astralo** [structurizr/java#296] [Q&A] [answered]
### Question

Is there a way to include software system to several group like this?
```
Test1 = softwaresystem "Test1" "Test1"
group "TestGroup1" {
    Test1 // attach system to the group
}
group "TestGroup2" {
    Test1 // attach system to the group
}
```

I trying to include all systems into one dsl file and use !include for including to workspace.
But dsl doesn't allow me to use variables with system like Test1 in groups

I have lots of services and including all of them to groups is not convenient unfortunately

**Comment by simonbrowndotje** [comment, marked as answer]
A software system can only be included in a single group, so I'm afraid not.
